tests/profiler/user_workflow_fixtures.py

```diff
@@ -0,0 +1,194 @@
+from typing import List
+
+import pytest
+
+from great_expectations.core import ExpectationConfiguration, ExpectationSuite
+
+# TODO: Move these fixtures to integration tests
+
+@pytest.fixture(scope="module")
+def alice_columnar_table_single_batch():
+    profiler_config = """
+variables:
+  max_user_id: 999999999999
```

Comment:
Do variables have the dollar sign in front?

Reply:
Absolutely! When referenced elsewhere, but not in the definition.